RM-925 RN 2.19.0 (#519)
percona/pmm-doc: docs/release-notes/2.19.0.md

    <col class="field-name">
    <col class="field-body">
  </colgroup>
  <tbody valign="top">
    <tr class="field-odd field">
      <th class="field-name">Date:</th>
      <td class="field-body">June 29, 2021</td>
    </tr>
    <tr class="field-even field">
      <th class="field-name">Installation:</th>
      <td class="field-body">
        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
    </tr>
  </tbody>
</table>

Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.

## Release Highlights

- Backup Management can now be turned on from the UI. Go to *Settings-->Advanced settings*, and you will see it in the *Technical Preview* section. We also added support of MongoDB services on-demand backup and restore. For now, it only supports ReplicaSet on S3-compatible storage.

- Dashboards improvements

    - There are several community-driven improvements to ProxySQL data collection, with new dashboards to expose such metrics like: *Queries Latency histograms* and `SHUNNED_REPLICATION_LAG` state.

>     - Fixes for Amazon Aurora service detection on dashboards (LINK), MongoDB ReplicaSet Summary, and other MongoDB memory-related panels.

Reviewer: ```suggestion
    - Fixes for Amazon Aurora service detection on [the dashboard](https://pmmdemo.percona.com/graph/d/mysql-amazonaurora/mysql-amazon-aurora-details?var-service_name=rds-aurora57-instance-1), MongoDB ReplicaSet Summary, and other MongoDB memory-related panels.
```

> - Improvements to the security of DBaaS secrets by generating passwords for installed operators. We shipped Automated operator installation in provided K8s cluster in PMM 2.18, and here we are improving its security.  (???)

Reviewer 2: ```suggestion
- Improvements to DBaaS secrets by generating strong passwords for operators. This is an improvement to the Automated Operator Installation released in PMM 2.18, which will greatly enhance security.  
```

## New Features

* [PMM-7639](https://jira.percona.com/browse/PMM-7639): Backup Management: Ability to remove Backup Location even if there are some backup artifacts on it

> * [PMM-7568](https://jira.percona.com/browse/PMM-7568): Backup Management: Simple restore for MongoDB ReplicaSet
> * [PMM-7567](https://jira.percona.com/browse/PMM-7567): Backup Management: Simple backup for MongoDB ReplicaSet

Reviewer 2: ```suggestion
* [PMM-7567](https://jira.percona.com/browse/PMM-7567): Backup Management: Simple backup for MongoDB ReplicaSet
* [PMM-7568](https://jira.percona.com/browse/PMM-7568): Backup Management: Simple restore for MongoDB ReplicaSet
```

## Improvements

> * [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present on dashboard Queries Latency histograms from ProxySQL

Reviewer 2: ```suggestion
* [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present histograms from ProxySQL on Queries Latency dashboard
```

> * [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present histograms from ProxySQL on Queries Latency dashboard

Reviewer: ```suggestion
* [PMM-8112](https://jira.percona.com/browse/PMM-8112): Dashboards: Collect and present histograms from ProxySQL on Queries Latency dashboard (Thanks to [foosin](https://github.com/foosinn) for helping with this improvement )
```

> * [PMM-8081](https://jira.percona.com/browse/PMM-8081): Dashboards: Add collection and presentation for `SHUNNED_REPLICATION_LAG` state to `proxysql_connection_pool_status`

Reviewer: ```suggestion
* [PMM-8081](https://jira.percona.com/browse/PMM-8081): Dashboards: Add collection and presentation for `SHUNNED_REPLICATION_LAG` state to `proxysql_connection_pool_status` 
(Thanks to [spaceform](https://github.com/spaceform) for helping with this improvement )
```

* [PMM-7584](https://jira.percona.com/browse/PMM-7584): Components Upgrade: VictoriaMetrics 1.53.1 to 1.60.0 TBD (link for VM features)

> * [PMM-8001](https://jira.percona.com/browse/PMM-8001): Better error handling when `pg_stat_monitor` has an unsupported version

Reviewer 2: ```suggestion
* [PMM-8001](https://jira.percona.com/browse/PMM-8001): Better error handling when `pg_stat_monitor` is an unsupported version
```

* [PMM-7659](https://jira.percona.com/browse/PMM-7659): DBaaS: Ability to specify the type of connection for DBaaS cluster during DB Cluster creation
* [PMM-7828](https://jira.percona.com/browse/PMM-7828): DBaaS: Select Database Type by default if only one operator is installed

> * [PMM-8153](https://jira.percona.com/browse/PMM-8153): Backup Management: Improve UX for Backup lists when Service acceptable for restore is removed

Reviewer 2: I have no idea what this means?  Does this mean when there is no acceptable restore service/location available?

Author: ```suggestion
[PMM-8153](https://jira.percona.com/browse/PMM-8153): Backup Management: Disable 'Restore' button for backups whose service has been deleted
```

Author: @rnovikovP Is this ok?

## Bugs Fixed

> * [PMM-7194](https://jira.percona.com/browse/PMM-7194): 'Share with Percona' option doesn't export all panels

Reviewer 2: ```suggestion
* [PMM-7194](https://jira.percona.com/browse/PMM-7194): 'Share with Percona' option doesn't export data from collapsed panels
```

* [PMM-7104](https://jira.percona.com/browse/PMM-7104): Slowlog rotation by pmm-agent causing additional unexpected rotation of binary logs (Thanks to Arthur Tokarchuk for reporting this issue)
* [PMM-8125](https://jira.percona.com/browse/PMM-8125): Error of monitoring GCP based PostgreSQL because of internal database `cloudsqladmin`
* [PMM-6778](https://jira.percona.com/browse/PMM-6778): Can't specify custom labels during node addition/configuration

> * [PMM-8090](https://jira.percona.com/browse/PMM-8090): Multi-request protection breaks metrics gathering

Reviewer: ```suggestion
* [PMM-8090](https://jira.percona.com/browse/PMM-8090): Multi-request protection breaks metrics gathering (Thanks to [Francisco Miguel Biete](https://github.com/fmbiete) for fixing this bug )
```
